docs/changelog.rst

```diff
@@ -10,18 +10,24 @@ Changes will be added here periodically from the "Suggested changelog entry"
 block in pull request descriptions.
 
 
-
 IN DEVELOPMENT
 --------------
 
 Changes will be summarized here periodically.
 
-Version 2.10.1 (Oct 2?, 2022)
+Version 2.10.1 (Oct 24, 2022)
 -----------------------------
 
+There is some concern about ABI compatibility between 2.10.x and previous
+versions, possibly related to exception handling. Please test if this is
+important to you before upgrading, and report an issue or comment on an issue
+if there's a problem. We will be bumping the internal ABI version in an
+upcoming release.
 
-Changes:
+Using at least this version is recommended if you need Python 3.11 support
```

Comment:
Suggested change:
________
This release has one change (PR #4119) that improves compatibility with Python 3.11 for users of pybind11/embed.h.
________

I think it's important to highlight, but I'd let people decide by themselves how important this is to them. (I assume it's not important to most users.)

Reply:
Reworded to clarify that this the first version to support embedding 3.11.